src/pages/Booking/components/Theater/shared/TheaterWrappr.jsx

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+const TheaterWrappr = styled.div`
```

Comment:
```suggestion
const TheaterWrapper = styled.div`
```
작은 오타지만 이런 부분들이 전부 휴먼 에러로 이어질 수 있기 때문에 잘 잡아주셔야 합니다. 오타 관련 익스텐션도 있으니 한번 찾아보세요.